src/pages/latestprojects/Components/Noteslist.tsx

```diff
@@ -0,0 +1,22 @@
+import { Event } from "nostr-tools";
+import NoteCard from './NoteCard.tsx';
+
+interface Props {
+  projects: Event[];
+}
+
+export default function Noteslist({ projects }: Props) {
+  const sortedProjects = projects.slice().sort((a, b) => {
+    // Sorting logic here if needed
+  });
+
+  return (
+    <div className="notes-list-container" style={{ margin: '0 20px', padding: '10px' }}>
+      <div className="flex" style={{ display: 'flex', flexDirection: 'column' }}>
+        {sortedProjects.map((nostr, index) => (
+          <NoteCard key={index} nostrich={nostr.pubkey} content={nostr.content} />
+        ))}
+      </div>
+    </div>
+  );
+}

```

Comment:
we don't really use classNames, and I don't think there are classes for notes-list-container or flex in the platform, let's stick to Chakra elements here, and use parameters instead of styles inside `style parameter`
This is important because only chakra elements are themed automatically, the default tags like div span are not themed.